src/veh_interact.cpp

```diff
@@ -1802,19 +1850,33 @@ void veh_interact::display_stats()
             y[i] += hoff - 1;
         }
     }
+    if(most_repariable != -1 && most_repariable != mostDamagedPart){
+        mvwprintz(w_stats, y[7], x[7], c_ltgray, _("Needs repair:"));
+        const auto iw = utf8_width(_("Needs repair:")) + 1;
+        x[7] += iw;
+        w[7] -= iw;
+        const auto &pt = veh->parts[most_repariable];
+        const auto hoff = fold_and_print( w_stats, y[7], x[7], w[7],
+                                          pt.is_broken() ? c_dkgray : pt.base.damage_color(), pt.name() );
+        // If fold_and_print did write on the next line(s), shift the following entries,
+        // hoff == 1 is already implied and expected - one line is consumed at least.
+        for( size_t i = 8; i < sizeof(y) / sizeof(y[0]); ++i) {
+            y[i] += hoff - 1;
+        }
```

Comment:
This snippet looks suspiciously similar to the one above. Could be slightly de-duplicated.